### ISSUE TITLE
News pages feedback

### ISSUE THREAD

**Issue by JoshHeng** [UWCS/new-site#39]
- [x] 1. On mobile, the horizontal margins is quite high and could probs be reduced
![image](https://github.com/UWCS/new-site/assets/34946235/01eacd37-6bf1-488d-9f06-391249ade1f4)

- [x] 2. I'm not sure why all the news items say Current News at the end of the description?
![image](https://github.com/UWCS/new-site/assets/34946235/a5762ac5-1110-4517-bce8-6422d53d69a7)

- [x] 3. I would maybe rephrase this as 'Back to Current News' or 'View Current News' 
![image](https://github.com/UWCS/new-site/assets/34946235/6f581f62-3696-4151-917a-66e2d06719b0)

- [x] 4. Is it intentional that the old news articles have not been migrated yet?

- [x] 5. Open graph title and description for the meeting minutes have trailing/leading whitespace
![image](https://github.com/UWCS/new-site/assets/34946235/7bf1eeca-5b8a-4eb3-bf37-946d4ab9b1aa)

**Comment by ericthelemur** [comment]
1 still to do tomorrow
2 was a painful fix (the link was in the section's content, but i assumed zola's usual summary selection would work, but it apparently doesn't apply to sub sections (yay). link is now in the template)
3 & 5 done
4 oops forgot to update build script when switching from enabling the archive by default to disabling it

**Comment by Mole1424** [comment]
all done
